tests/test-karma.js

```diff
@@ -1,7 +1,7 @@
 /**
  * Karma test support.
  *
- * Copyright (c) 2011-2020 Digital Bazaar, Inc. All rights reserved.
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
  */
 // load polyfills
 // no polyfills are currently required.
```

Comment:
Can remove this file since it was removed from karma conf and isn't loading polyfills in this project.